integration-tests/const.py

```diff
@@ -14,6 +14,7 @@
 
 NS = {'d': 'http://maven.apache.org/POM/4.0.0'}
 ZIP_FILE_EXTENSION = ".zip"
+WSO2SERVER='bin/integrator.sh'
```

Comment:
Since we have two spaces between `=` character and also we have used double-quotes in other constants declarations, shall we do the same here (to maintain a single formatting)?